Tools/LyTestTools/ly_test_tools/_internal/managers/artifact_manager.py

```diff
@@ -48,7 +48,7 @@ def _get_collision_handled_filename(self, file_path, amount=1):
             updated_path = f"{file_without_ext}_{i}{extension}"
             if not os.path.exists(updated_path):
                 return updated_path
-        logger.info(f"Maximum number of attempts: {amount} met when trying to handle name collision for file: "
```

Comment:
Since this is destructive, I suggest it should actually be warning (higher than both debug and info). Intended to be uncommon, so I don't expect it is contributing to the current spamminess